client/src/containers/Dispositif/Dispositif.js

```diff
@@ -172,6 +172,9 @@ export class Dispositif extends Component {
     this.checkUserFetchedAndInitialize();
     window.scrollTo(0, 0);
     // this._initializeDispositif(this.props);
+    if (!isMobile) {
+      window.addEventListener("scroll", () => this.handleScroll());
```

Comment:
tu peux mettre this.handleScroll au lieu de ()=>this.handleScroll() (tu as juste besoin de donner la reference de al fonction)